roles/container-engine/docker/tasks/main.yml

```diff
@@ -144,28 +108,11 @@
     - ansible_distribution in ["CentOS","RedHat"] and not is_ostree
     - yum_result.results | length == 0
 
-- name: Copy yum.conf for editing
-  copy:
-    src: "{{ yum_conf }}"
-    dest: "{{ docker_yum_conf }}"
-    remote_src: yes
-  when: ansible_distribution in ["CentOS","RedHat","OracleLinux"] and not is_ostree
```

Comment:
ditto: Why can we remove this task?

Reply:
Commit message:
docker: remove obsoletes=0 in yum.conf

This was introduced in ef7f5ed (2018)
obsoletes=0 is not present in the official repo config
https://download.docker.com/linux/centos/docker-ce.repo
so it might not be needed for some time

And also CI pass just fine ;)